tm/src/main/java/org/apache/seata/tm/api/DefaultGlobalTransaction.java

```diff
@@ -113,6 +113,8 @@ public void begin(int timeout, String name) throws TransactionException {
         xid = transactionManager.begin(null, null, name, timeout);
         status = GlobalStatus.Begin;
         RootContext.bind(xid);
+        RootContext.setTimeout(timeout);
+        RootContext.setBranchBeignTime(createTime);
```

Comment:
If I set this value here I think it is ambiguous, the time here should be the start time of the global transaction and not the begin time of the branch transaction.